src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --link-color: #00ACDC;
+  --header-height: 60px
```

Comment:
The CSS custom property '--header-height' is missing a semicolon at the end of the declaration. It should be '--header-height: 60px;'. Missing semicolons can lead to unexpected behavior in CSS.